index.js

```diff
@@ -1,11 +1,5 @@
 'use strict';
+const getFromEnv = parseInt(process.env.ELECTRON_IS_DEV, 10) === 1;
+const isEnvSet = 'ELECTRON_IS_DEV' in process.env;
 
-const getFromEnv = () => {
-	if ('ELECTRON_IS_DEV' in process.env) {
-		return parseInt(process.env.ELECTRON_IS_DEV, 10) === 1;
-	}
-
-	return false;
-};
-
-module.exports = getFromEnv() || process.defaultApp || /[\\/]electron-prebuilt[\\/]/.test(process.execPath) || /[\\/]electron[\\/]/.test(process.execPath);
+module.exports = isEnvSet ? getFromEnv : (process.defaultApp || /[\\/]electron-prebuilt[\\/]/.test(process.execPath) || /[\\/]electron[\\/]/.test(process.execPath));
```

Comment:
Unrelated, but just a thought, since I have you here. Maybe we should change`/[\\/]electron[\\/]/` to `/node_modules[\\/]electron[\\/]/` to fix #4?